### PR TITLE
Make possible OpenID auth via POST

### PR DESCRIPTION
**Pull request by zorn-v**
Wargaming OpenID do post request for example

### CODE REVIEW

File: src/Adapter/OpenID.php [hunk 1]
```diff
@@ -82,7 +82,7 @@ public function authenticate()
             return true;
         }
 
-        $openid_mode = filter_input(INPUT_GET, 'openid_mode');
+        $openid_mode = filter_input($_SERVER['REQUEST_METHOD'] === 'POST' ? INPUT_POST : INPUT_GET, 'openid_mode');
```
Comment (ApacheEx) [inline src/Adapter/OpenID.php]: what if use `$_REQUEST` superglobal array which contains GET + POST?

```php
if (!empty($_REQUEST['openid_mode'])) {
    $this->authenticateBegin();
}
else {
    ...
}
```

Comment (zorn-v) [reply src/Adapter/OpenID.php]: I think you meant `empty($_REQUEST['openid_mode'])`
At all without difference, but I just left `filter_input` as it was

Comment (ApacheEx) [reply src/Adapter/OpenID.php]: yeah, 💯  (updated)

can you update this PR according to my comment?

**Comment by ApacheEx** [comment]
Many thanks 👍

**Pull request merged**